docs/developer/style.md

```diff
@@ -0,0 +1,215 @@
+# Go Style Guide in Vald
+
+## Table of Contents
+
+- [Introduction](#Introduction)
+- [Style](#Style)
+- [Test](#Test)
+	- [Table-Driven-Test](#"The guideline of Table-Driven-Test for Vald")
+
+## Introduction
+
+## Style
+
+## Test
+
+This section will describe the rule for writing test code in Vald.
+
+### The guideline of Table-Driven-Test for Vald
```

Comment:
Oh, did you say `Table-Driven-Test` is better?  you said it need not to add `guideline` each section before because this document describes the guideline, right?

Reply:
I`m sorry :bow:
I fixed but I forgot commit....